website/blog/2025-12-05-kubernetes-ai-conformance-aks/index.md

```diff
@@ -0,0 +1,188 @@
+
+The program is developed in the open by the [WG AI Conformance](https://github.com/kubernetes/community/tree/master/wg-ai-conformance) working group. This group operates under an openly published charter and focuses on:
+
+- Defining a reference architecture for AI workloads on Kubernetes
+- Establishing framework support requirements
+- Creating test criteria for key capabilities such as GPU integration, volume handling, and job-level networking
+
+## Why AI Conformance matters
+
+Running AI workloads on Kubernetes introduces unique challenges. Teams often struggle with GPU driver compatibility, specialized scheduling requirements for distributed training, and the complexity of exposing inference endpoints at scale. Without a common baseline, organizations risk building on fragmented, vendor-specific implementations that limit flexibility.
+
+According to [Linux Foundation Research on Sovereign AI](https://www.linuxfoundation.org/hubfs/Research%20Reports/lfr_sovereign_ai_090525a.pdf), 82% of organizations are already building custom AI solutions, and 58% use Kubernetes to support those workloads. With 90% of enterprises identifying open-source software as critical to their AI strategies, the risk of fragmentation and inconsistent performance is rising. The Kubernetes AI Conformance Program responds directly to this need.
+
+The AI Conformance Program addresses these challenges by establishing a verified set of capabilities that every conformant platform must support. When you deploy on AKS, a certified AI-conformant platform, you benefit from:
+
+- **Predictable scaling**: Your training jobs and inference services scale consistently using standardized APIs and autoscaling behaviors
+- **Hardware optimization**: GPU and accelerator resources are managed through proven Kubernetes primitives, maximizing utilization
+- **Workload mobility**: Applications built for one conformant platform work on any other, reducing vendor lock-in
+- **Ecosystem compatibility**: Popular ML frameworks, operators, and tools function reliably because they target a known, tested baseline
+
+## Key requirements for AI Conformance
+
+The working group maintains a versioned specification that aligns with Kubernetes releases. Each requirement goes through a graduation process, starting as a SHOULD recommendation and eventually becoming a MUST for certification. The requirements are organized into categories: accelerators, networking, scheduling and orchestration, observability, security, and operators.

```

Comment:
```suggestion
The Kubernetes AI Conformance program maintains a versioned specification for each Kubernetes release starting with v1.33. Each requirement goes through a graduation process, starting as a SHOULD recommendation and eventually becoming a MUST for certification. 
```
Suggest to remove those categories since they might change over time.